install/kubernetes/templates/istio-sidecar-injector-configmap-release.yaml.tmpl

```diff
@@ -0,0 +1,107 @@
+kind: ConfigMap
+metadata:
```

Comment:
Also for a future PR - it may be worth allowing developer to select with an annotation if they want debug or release sidecar, and to use an {{ if }} to select it instead of 2 templates. 
I'm not very sure how one will be selected - is it the global config ?

Reply:
Sounds reasonable. See https://github.com/istio/istio/issues/2790#issuecomment-360325635 for an example of per-pod overrides. A similar idea could be applied for debug vs release (or even more selectively, e.g. proxy image, log level, core dumps.